content/en/docs/tasks/run-application/run-replicated-stateful-application.md

```diff
@@ -206,15 +207,15 @@ server, and an `xtrabackup` container that acts as a
 [sidecar](https://kubernetes.io/blog/2015/06/the-distributed-system-toolkit-patterns).
 
 The `xtrabackup` sidecar looks at the cloned data files and determines if
-it's necessary to initialize MySQL replication on the slave.
+it's necessary to initialize MySQL replication on the replica.
 If so, it waits for `mysqld` to be ready and then executes the
 `CHANGE MASTER TO` and `START SLAVE` commands with replication parameters
 extracted from the XtraBackup clone files.
 
-Once a slave begins replication, it remembers its MySQL master and
+Once a replica begins replication, it remembers its primary MySQL server and
```

Comment:
> Once a replica begins replication

It's not wrong but it's a bit clunky. Optionally, rewrite the whole sentence to avoid this.

**I'd be happy to see this merge with the clunk left in, and then worry about the nits later.**

Reply:
Yeah it is clunky. I'm starting to remember why I prefer writing code to documentation.

Reply:
I do get that, devries. Writing doc can be tricky and appear to be nit-picky. But I am asked to document decisions by Kubernetes as to which terms will replace master/slave, and in what contexts--so that we can get _our_ doc right. So for now my question is:  What sort of objects/processes will the terms **primary** and **replica** apply to, if this pull request is accepted? These terms are technically noun _modifiers_ (adjectives), so I need a noun. Are we talking about processes, databases, files? Looks like files, is that correct? Is this request only for yaml files at this point?

Reply:
In this case we are talking about database servers. I have been using "server" as the associated noun in these instances.

Reply:
Thank you very much!

Reply:
I'm also happy to see this clunk merge in.